### PR TITLE
Added FreeBSD depext for emacs

### PR DESCRIPTION
**Pull request by mtelvers**

### CODE REVIEW

**Comment by haochenx** [comment]
same as https://github.com/ocaml/opam-repository/pull/24425#issuecomment-1718104677: you can try add an `x-ci-accept-failures` field.

**Comment by tmcgilchrist** [comment]
I redeployed opam-repo-ci with https://github.com/ocurrent/opam-repo-ci/pull/236 and https://github.com/ocurrent/opam-repo-ci/pull/231. Now it's only failing on macOS but installing locally on macOS Ventura works. It needs more investigation on why it is failing.

File: packages/conf-emacs/conf-emacs.1/opam [hunk 1]
```diff
@@ -14,6 +14,7 @@ depexts: [
   ["emacs-nox"] {os-family = "suse" | os-family = "opensuse"}
   ["emacs-nox"] {os-distribution = "ol"}
   ["emacs"] {os-distribution = "arch"}
+  ["emacs"] {os = "freebsd"}
```
Comment (tmcgilchrist) [inline packages/conf-emacs/conf-emacs.1/opam:17]:
```suggestion
  ["emacs"] {os = "freebsd"}
  ["emacs"] {os-distribution = "homebrew" & os = "macos"}
```
🤦🏻 maybe adding a line for macOS will help.

Comment (tmcgilchrist) [reply packages/conf-emacs/conf-emacs.1/opam:17]: ^ I've tested that on the CI cluster for macOS x86 and it works.

**Comment by tmcgilchrist** [comment]
There is a FreeBSD health-check running on http://freebsd-health-check.ocamllabs.io:8080 validating 4.14 and 5.1. @avsm

**Pull request merged**